src/tools/kdb/command.h

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ */
+
+#ifndef COMMAND_H
+#define COMMAND_H
+
+#include <kdb.h>
+
+#define ADD_BASIC_OPTIONS(baseSpec, baseKeyName)                                                                                           \
+	ksAppendKey (baseSpec, keyNew (baseKeyName "/debug", KEY_META, "description",                                                      \
+				       "Give debug information or ask debug questions (in interactive mode).", KEY_META, "opt", "d",       \
+				       KEY_META, "opt/long", "debug", KEY_META, "opt/arg", "none", KEY_END));                              \
+	ksAppendKey (baseSpec, keyNew (baseKeyName "/verbose", KEY_META, "description", "Explain what is happening.", KEY_META, "opt",     \
+				       "v", KEY_META, "opt/long", "verbose", KEY_META, "opt/arg", "none", KEY_END));                       \
+	ksAppendKey (baseSpec, keyNew (baseKeyName "/version", KEY_META, "description", "Print version info.", KEY_META, "opt", "V",       \
+				       KEY_META, "opt/long", "version", KEY_META, "opt/arg", "none", KEY_END));                            \
+	ksAppendKey (baseSpec, keyNew (baseKeyName "/profile", KEY_META, "description", "Use a different profile for kdb configuration.",  \
+				       KEY_META, "opt", "p", KEY_META, "opt/arg/help", "NAME", KEY_META, "opt/long", "profile", KEY_META,  \
+				       "opt/arg", "required", KEY_END));                                                                   \
+	ksAppendKey (baseSpec, keyNew (baseKeyName "/color", KEY_META, "description", "Print never/auto(default)/always colored output.",  \
+				       KEY_META, "opt", "c", KEY_META, "opt/arg/help", "WHEN", KEY_META, "opt/long", "color", KEY_META,    \
+				       "opt/arg", "required", KEY_END));                                                                   \
+	ksAppendKey (baseSpec,                                                                                                             \
+		     keyNew (baseKeyName "/nonewline", KEY_META, "description", "Suppress the newline at the end of the output.",          \
+			     KEY_META, "opt", "n", KEY_META, "opt/long", "no-newline", KEY_META, "opt/arg", "none", KEY_END));
+
+
+#define GET_OPT(options, key) keyString (ksLookupByName (options, "proc:" key, 0))
```

Comment:
> Only cascading lookups should be needed.

With the current setup it doesn't matter, since `elektraGetOpts` only produces `proc:/` keys. But with `gopts` using anything other than a cascading lookup here, is a straight-up bug.